app/main.py

```diff
@@ -22,6 +27,29 @@ class ConsumerError(Exception):
 
 
 class SeftConsumer:
+
+    @staticmethod
+    def extract_file(decrypted_payload, tx_id):
+        try:
+            file_contents = decrypted_payload['file']
+            file_name = decrypted_payload['filename']
+            case_id = decrypted_payload['case_id']
+            if not file_name or not file_contents or not case_id:
+                logger.error("Empty claims in message",
+                             file_name=file_name,
+                             file_contents="Encoded data" if file_contents else file_contents)
+                raise ConsumerError()
+            logger.debug("Decrypted file", file_name=file_name, tx_id=tx_id)
+            decoded_contents = base64.b64decode(file_contents)
+            return decoded_contents, file_name, case_id
+        except (KeyError, ConsumerError) as e:
+            logger.error("Required claims missing quarantining message",
+                         exception=e,
+                         keys=decrypted_payload.keys(),
```

Comment:
decrypted_payload is a dictionary object, keys is just the dict built in method.